db/seeds.rb

```diff
@@ -1,26 +1,26 @@
 
-Issue.create(topic: "Environment", description: "Set to be a hotly contested issue, candidates will debate whether climate change is a hoax or if the world is set to end in 20 years time.", featured: true, picture: 'environment')
+Issue.create(topic: "Environment", description: "Set to be a hotly contested issue, candidates will debate whether climate change is a hoax or if the world is set to end in 20 years time. It still remains a mystery.", featured: true, picture: 'environment')
 
-Issue.create(topic: "Immigration", description: "Some candidates insist that their families materialized from America's soil, never to have experienced immigration, while others would like to open Lady Liberty's arms to the masses.", featured: true, picture: 'immigration')
+Issue.create(topic: "Immigration", description: "Some candidates insist their families materialized from America's soil, never to have experienced immigration, while others would like to open Lady Liberty's arms to the masses.", featured: true, picture: 'immigration')
 
-Issue.create(topic: "ISIS", description: "We can already hear the arguments unfolding and the debates haven't started. Republican: 'In conjunction with sharing our deep held values of A-MUR-I-CA, cheeseburgers included, we are making great progress against ISIS with every drone strike. They're the best!' Democrats: 'No no no, drone attacks are the work of the devil! We should sit down to a nice lunch with their leader and discuss these issues like adults.'", picture: 'isis')
+Issue.create(topic: "ISIS", description: "We can already hear the arguments unfolding and the debates haven't even started. Republicans: 'In conjunction with sharing our deep held values of A-MUR-I-CA, cheeseburgers included, we are making great progress against ISIS with every drone strike. They're the best!' Democrats: 'No no no, drone attacks are the work of the devil! We should sit down to a nice lunch with their leaders and discuss these issues like adults.'", picture: 'isis')
 
 Issue.create(topic: "Gun Control", description: "'Let them have guns!,' some politicians might yell, channeling their inner Marie Antoinettes, while others will argue that prison rules should apply - anything that can be used as a weapon, should be illegal. Please ask yourself, would you like to own an AK47 or would you like to eat with your hands?", picture: 'guns')
 
-Issue.create(topic: "Economy", description: "The hard working, and deserving, 1 per-centers are the way of the future according to some. Others envision a single world commune where everyone's opinion is equally important.", featured: true, picture: 'economy')
+Issue.create(topic: "Economy", description: " According to some, the hard working and deserving 1 per-centers are the way of the future. Others, envision a single world commune where everyone's opinion is equally important.", featured: true, picture: 'economy')
```

Comment:
Line is too long. [261/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.